src/components/CategoricalDisplay.jsx

```diff
@@ -103,7 +103,7 @@ const CategoricalDisplay = ({
     return (
         <div className="cq-categorical-display">
             {renderMobileTitle()}
-            <div className={`cq-lookup-filters ${hideLookupFilter ? 'hide' : ''}`}>
+            <div className={`cq-lookup-filters ${hideLookupFilter ? 'scroll-down' : ''}`}>
```

Comment:
@MahboobehMohammadi the renaming should include `hideLookupFilter` as well, since we handle the logic to hide it in CSS. Think about someone reading this code in the future. He would be wondering "You want to hide the lookup filter, but you put a class `scroll-down`?".

In short, when ppl read the code in future, if the parts are related, it must read as accordingly.

Reply:
@brucebinary Since the class name became `scroll-down` i think ii should use something like `isScrollingDown` or st like that.

Reply:
sounds good